src/style.css

```
@@ -0,0 +1,64 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.head {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 40px 0 50px;
```

```suggestion
  padding: 0 50px;
```
Why 40px? We have 50px both to the left and to the right